src/components/Toolbar.tsx

```diff
@@ -30,7 +30,7 @@ interface ToolbarProps {
 }
 
 /** Toolbar component. */
-const Toolbar: FC<ToolbarProps> = ({ customize, onSelect, selected }) => {
+const Toolbar: FC<ToolbarProps> = ({ customize, onSelect, selected, handleMouseUp }) => {
```

Comment:
Remove unused prop

Reply:
That prop is used. Removing it throws an error `handleMouseUp is not defined`

Reply:
Sorry, I got it. Found the issue.